README.md

```diff
@@ -180,10 +180,74 @@ curl -X 'GET' \
   --output ih9z9lwwv5r.zip
 ```
 
-# Manually uploading scans to Event Portal
+## Running a Discovery Scan and File Export Using the CLI
+
+A discovery scan and file export can be run as a single command from a terminal using the event-management-agent CLI. The CLI can be accessed via either the Event Management Agent jar file or a docker container using the instructions below. Use the `Upload Scan File` mode when [creating the Event Management agent in Event Portal](https://docs.solace.com/Cloud/Event-Portal/event-portal-collect-runtime-data.htm#creating_connection_file) to create and download the standalone connection file. Set the resource passwords as environment variables that match the variables set in the connection file.
+
+### Discovery Scan and File Export from CLI Using the Jar File
+
+The Event Management Agent jar file can be obtained by either:
+* downloading the file from the [releases page](https://github.com/SolaceProducts/event-management-agent/releases). A release of `1.5.0` or greater is required.
+* running a [build of the source code](#installing-maven-dependencies-and-building-the-event-management-agent-jar-file)
+
+To run a scan:
+* provide the id of the event broker specified in the connection file. (e.g. mysolacebroker)
+* provide the event-management-agent jar file. (e.g. event-management-agent-1.5.0-SNAPSHOT.jar)
+* provide the path and filename for the resulting scan data zip file. (e.g. /tmp/scan.zip)
+* provide the location of the connection file (e.g. /path/to/file/AcmeRetailStandalone.yml)
+
+The command is as follows:
+`java -jar event-management-agent-1.5.0-SNAPSHOT.jar scan mysolacebroker /path/to/scan.zip --springdoc.api-docs.enabled=false --spring.main.web-application-type=none --spring.config.location=/path/to/file/AcmeRetailStandalone.yml`
+
+### Discovery Scan and File Export From CLI Using Docker
+
+To run a scan and export operation using the CLI from docker:
+* provide the id of the event broker specified in the connection file. (e.g. mysolacebroker)
+* provide the path and filename for the resulting scan data zip file. (e.g. /tmp/scan.zip)
+* provide the location of the connection file (e.g. /path/to/file/AcmeRetailStandalone.yml)
+* provide the name of the Event Management Agent container (e.g. event-managenement-agent)
+* provide the Event Management Agent Docker image (e.g. solace/event-management-agent:latest)
+* add all password environment variables to the environment of the docker container (example BROKER_PASSWORD)
+
+The commands are as follows:
+`CMD_LINE_ARGS="scan abcdefg123 /tmp/scan.zip  --springdoc.api-docs.enabled=false --spring.main.web-application-type=none"`
+`run -d -v /path/to/file/AcmeRetailStandalone.yml:/config/ema.yml -v /path/to/scandir:/tmp  --env CMD_LINE_ARGS=${CMD_LINE_ARGS} --env BROKER_PASSWORD=${BROKER_PASSWORD} --name event-management-agent event-management-agent:latest`
+
+## Running a Discovery File Upload Using the CLI
+
```

Comment:
@AndreaKelso I was missing the word "upload" in the first sentence. Hopefully it's clear now.